static/app/utils/replays/hydrateFrames.tsx

```diff
@@ -22,6 +22,10 @@ export default function hydrateFrames(attachments: unknown[]) {
       return;
     }
     if (isBreadcrumbFrameEvent(attachment)) {
+      if (attachment.data.payload.category === 'sentry.feedback') {
+        // @ts-expect-error In SDK <= 7.100.0 we were incorrectly setting the timestamp
+        attachment.data.payload.timestamp = attachment.data.timestamp;
+      }
```

Comment:
@billyvg we could also just drop this, feedback isn't GA so older sdk's might not matter so much.

Reply:
Since this has been in the SDK for awhile, let's keep it so that our early adopters can see it.